e2e/tests/i18n/editview.spec.ts

```diff
@@ -0,0 +1,219 @@
+import { test, expect } from '@playwright/test';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { login } from '../../utils/login';
+
+test.describe('Edit view', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('As a user I want to create a brand new document in the non-default locale', async ({
+    page,
+  }) => {
+    const LIST_URL = /\/admin\/content-manager\/collection-types\/api::product.product(\?.*)?/;
+    const EDIT_URL =
+      /\/admin\/content-manager\/collection-types\/api::product.product\/[^/]+(\?.*)?/;
+    const CREATE_URL =
+      /\/admin\/content-manager\/collection-types\/api::product.product\/create(\?.*)?/;
+
+    /**
+     * Navigate to our products list-view
+     */
+    await page.getByRole('link', { name: 'Content Manager' }).click();
+    await page.getByRole('link', { name: 'Products' }).click();
+    await page.waitForURL(LIST_URL);
+    await expect(page.getByRole('heading', { name: 'Products' })).toBeVisible();
+
+    /**
+     * Assert we're on the english locale and our document exists
+     */
+    await expect(page.getByRole('combobox', { name: 'Select a locale' })).toHaveText(
+      'English (en)'
+    );
+    await expect(
+      page.getByRole('row', { name: 'Nike Mens 23/24 Away Stadium Jersey' })
```

Comment:
😆 the files I wrote are becoming such a let down, I need to up my game.
Football is life!